backend/src/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export enum GameState {
+  IN_PROGRESS = 'in_progress',
+  FINISHED = 'finished'
+}
+
+// for destination, I'm assuming it will be a link
+// I'll change it if needed
+export interface ButtonCoordinates {
+  x: number;
+  y: number;
+  destination: string;
+}
+
+export interface Level {
+  photoLink: string;
+  locationName: string;
+  locationZCoordinate: number;
+  buttons?: ButtonCoordinates[];
+}
+
+export interface Game {
+  status: GameState;
+  levels: Level[];
+  score: number;
+  owner: string;
+  startTime: number; // To calculate time bonuses?
```

Comment:
this whole interface looks great, id just recommend using a Date type for startTime and similar to the level thingy i mentioned before consider using `firebase.firestore.DocumentSnapshot.id` as the owner type instead of string and also add an id here to reference the db instance of game